frontend/components/TableContainer/DataTable/TooltipTruncatedTextCell/TooltipTruncatedTextCell.tsx

```diff
@@ -9,14 +9,16 @@ import { COLORS } from "styles/var/colors";
 interface ITooltipTruncatedTextCellProps {
   value: React.ReactNode;
   /** Tooltip to dispay. If this is provided then this will be rendered as the tooltip content. If
-   * not the value will be displayed as the tooltip content. Defaults to `undefined` */
+   * not, the value will be displayed as the tooltip content. Defaults to `undefined` */
   tooltip?: React.ReactNode;
   /** If set to `true` the text inside the tooltip will break on words instead of any character.
    * By default the tooltip text breaks on any character.
    * Default is `false`.
    */
   tooltipBreakOnWord?: boolean;
+  /** DEPRICATED: use the prop `className` in order to add custom classes to this component */
```

Comment:
typo here, as a small suggestion, using JSDoc should show a warning in most text editors (not all)

![image](https://github.com/fleetdm/fleet/assets/4419992/55a3721e-0ada-4796-9900-8eafe34f29f8)


```suggestion
  /** @deprecated use the prop `className` in order to add custom classes to this component */
```

Reply:
nice thanks.